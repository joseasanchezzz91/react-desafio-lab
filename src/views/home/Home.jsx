import React, { Component } from "react";
import List from "../../components/list/List";


export default class Home extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div>
        <h1 className="text-center">Lista de Post</h1>
        <List />
      </div>
    );
  }
}
