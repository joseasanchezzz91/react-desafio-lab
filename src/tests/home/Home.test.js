import React from "react";
import { shallow } from "enzyme";
import Home from "../../views/home/Home";

describe("Pruebas en el componente Home", () => {
  let wrapper = shallow(<Home />);
  beforeEach(() => {
    wrapper = shallow(<Home />);
  });

  test("debe mostrar <Home/> correctamente", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("debe mostart H1 correctamente", () => {
    const h1 = "Lista de Post";
    const textH1 = wrapper.find("h1").text();
    expect(textH1).toBe(h1);
  });
});
