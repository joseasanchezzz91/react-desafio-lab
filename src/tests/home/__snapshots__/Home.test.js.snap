// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente Home debe mostrar <Home/> correctamente 1`] = `
<div>
  <h1
    className="text-center"
  >
    Lista de Post
  </h1>
  <List />
</div>
`;
